recipes/bioconductor-aldex2/meta.yaml

```diff
@@ -1,15 +1,19 @@
+{% set version="1.8.0" %}
+{% set name="ALDEx2" %}
+
 package:
-  name: bioconductor-aldex2
-  version: 1.8.0
+  name: 'bioconductor-{{ name|lower}}'
+  version: '{{ version }}'
 source:
-  fn: ALDEx2_1.8.0.tar.gz
+  fn: '{{ name }}_{{ version }}.tar.gz'
   url:
-    - http://bioconductor.org/packages/3.5/bioc/src/contrib/ALDEx2_1.8.0.tar.gz
-    - https://bioarchive.galaxyproject.org/ALDEx2_1.8.0.tar.gz
-    - https://depot.galaxyproject.org/software/ALDEx2/ALDEx2_1.8.0_src_all.tar.gz
-  md5: 979f4e257eb75248e8059a88674df0c8
+    - 'http://bioconductor.org/packages/3.5/bioc/src/contrib/{{ name }}_{{ version
+      }}.tar.gz'
+    - 'https://bioarchive.galaxyproject.org/{{ name }}_{{ version }}.tar.gz'
+    - 'https://depot.galaxyproject.org/software/{{ name }}/{{ name }}_{{ version }}_src_all.tar.gz'
+  sha256: 04c2bb73c634c3b274937dc3e7268efac9f08a93b8f0e091c7b5babeacd53610
 build:
-  number: 0
+  number: 2
```

Comment:
ok, everywhere :)

Reply:
whoops, forgot to `git checkout -- .` after the test. I'll fix all of these . . .